exampleMultipartyVideo/src/main/java/com/twilio/examplemultipartyvideo/MultiPartyActivity.java

```diff
@@ -410,6 +415,8 @@ private void intializeUI() {
         localVideoActionFab.setOnClickListener(localVideoClickListener());
         muteActionFab.show();
         muteActionFab.setOnClickListener(muteClickListener());
+        dominantSpeakerImg = new ImageView(this);
+        dominantSpeakerImg.setImageResource(android.R.drawable.ic_lock_silent_mode_off);
```

Comment:
Let's import [this icon](https://material.io/resources/icons/?icon=volume_up&style=baseline) from material and use as the icon.